internal/buildscripts/update-dep

```diff
@@ -2,28 +2,16 @@
 
 # Updates MODULE inside go.mod if it is already present to version VERSION.
 
+source $TOPDIR/internal/buildscripts/modules
+
+echo $modules
+
```

Comment:
Do you think this is useful to have, or was it just for debugging?

```suggestion
```

Reply:
Just for debugging 👍🏻